package.json

```diff
@@ -13,6 +17,10 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "babel-plugin-macros": "^3.0.1",
+    "ethers": "^5.0.32",
+    "graphql": "^15.5.0",
+    "luxon": "^1.26.0",
```

Comment:
what is luxon?

Reply:
It's the follow up lib to Moment.js .

I realized I was likely going to need to work with out Date object in a somewhat complex way, but I didn't get to implementing that just yet.